manifests/crd.yaml

```diff
@@ -1020,31 +1035,114 @@ spec:
           description: RestoreSpec contains the specification for a restore of a tidb
             cluster backup.
           properties:
-            backup:
-              description: Backup represents the backup object to be restored.
-              type: string
-            backupNamespace:
-              description: Namespace is the namespace of the backup.
+            backupPath:
+              description: BackupPath is the location of the backup.
               type: string
-            cluster:
-              description: Cluster represents the tidb cluster to be restored.
+            backupType:
```

Comment:
There is a `backupType` in `backupSpec`, why add one more here?

Reply:
This file is generated automatically